### PR TITLE
feat: Add marker CSS SVG properties

### PR DESCRIPTION
**Pull request by meyerweb**
### Description

Adds the properties `marker`, `marker-end`, `marker-mid`, and `marker-start`.

### Motivation

These properties have already been added to MDN reference content and need the formal syntax data to back them up.

### Related issues and pull requests

Relates to https://github.com/mdn/content/issues/34763.

### CODE REVIEW

**Comment by meyerweb** [comment]
I messed up my branch management, and thus this PR contains commits that are also in #755.  I hope that won’t screw up the process, but if so, let me know what you’d like me to do to fix it.

File: css/properties.json [hunk 1]
```diff
@@ -9245,6 +9361,22 @@
...
+    "percentages": "no",
+    "groups": [
+      "CSS Shapes"
+    ],
+    "initial": "auto",
+    "appliesto": "limitedSVGElementsShapes",
+    "computed": "asSpecified",
+    "order": "perGrammar",
+    "status": "standard",
+    "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/shape-outside"
```
Comment (wbamberg) [inline css/properties.json:9378]:
```suggestion
    "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/shape-rendering"
```

?

File: css/properties.json [hunk 2]
```diff
@@ -10065,6 +10213,22 @@
...
+  "vector-effect": {
+    "syntax": "none | non-scaling-stroke | non-scaling-size | non-rotation | fixed-position",
+    "media": "visual",
+    "inherited": false,
+    "animationType": "discrete",
+    "percentages": "no",
+    "groups": [
+      "Scalable Vector Graphics"
+    ],
+    "initial": "start",
```
Comment (wbamberg) [inline css/properties.json:10225]:
```suggestion
    "initial": "none",
```

?

**Review by wbamberg — CHANGES_REQUESTED** [review]

**Comment by estelle** [comment]
https://github.com/mdn/data/issues/750

**Pull request merged**